brambling/utils/model_tables.py

```diff
@@ -399,9 +399,10 @@ class AttendeeTable(CustomDataTable):
         ('Contact',
          ('email', 'phone')),
         ('Housing',
-         ('housing_status', 'housing_nights', 'housing_preferences',
-          'environment_avoid', 'environment_cause', 'person_prefer',
-          'person_avoid', 'other_needs')),
+         ('housing_status', 'housing_nights_if_needed',
+          'housing_preferences_if_needed', 'environment_avoid_if_needed',
+          'environment_cause_if_needed', 'person_prefer_if_needed',
+          'person_avoid_if_needed', 'other_needs_if_needed')),
```

Comment:
The field names should stay the same, since we don't need to distinguish between if_needed and if not needed. Simpler change. Also less likely to have unexpected side effects... in this case, for example, the old field names are still being used to determine what data to prefetch.